### ISSUE TITLE
Remote sources flash the menu

### ISSUE THREAD

**Issue by fbingha** [corejavascript/typeahead.js#138]
I would like help on a change so that when data is coming from remote sources, the menu doesn't flash as entry is made. Currently the menu is wiped out with every keypress and reshown. I would prefer the data itself to be updated without closing the menu between keypresses.

**Comment by LiamKarlMitchell** [comment]
Yeah that would be more ideal.

**Comment by vincenzo-antolini-os** [comment]
yeah, it is how it should be...

**Comment by codewiseio** [comment]
Yes I would like to see this also... Apparently this was working in version 0.10.2 before the fork but later reverted to the flickering behavior.

[#176](https://github.com/twitter/typeahead.js/issues/176)
[#1423](https://github.com/twitter/typeahead.js/issues/1423)

**Comment by KrisPetkov** [comment]
Is there any approach fixing this issue? Or some workaround?
I'm facing the same problem and it is quite annoying.
